payloads/remote_shell.py

```
@@ -0,0 +1,236 @@
+
+    # Read command line (1 byte at a time, buf1 es bytearray)
+    while True:
+        n = u64_to_i64(sc.syscalls.read(client_sock, buf1, 1))
+        log("read() returned %d" % n)
+
+        if n <= 0:
+            log("read <= 0, closing.")
+            sc.syscalls.close(client_sock)
+            raise SystemExit
+
+        # buf1[0] es un int 0–255; lo convertimos a char
```

Ruff (0.14.4)

Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)